test/util.py

```diff
@@ -0,0 +1,8 @@
+import os
+import uuid
+
+
+# TODO: use pytest.fixture to share this functionality with other tests
```

Comment:
In `pytest` way, this kind of function implemented as `@pytest.fixture` and use it in tests.
But the most gokart's tests are implemented in `unittest` way.
For consistency of codes, I currently implemented them in `unittest` way